recipes/antismash/meta.yaml

```diff
@@ -16,10 +15,10 @@ source:
 
 requirements:
   build:
-    - python
+    - python <3.0
```

Comment:
Thanks a lot for working on this @cmonjeau!
This pinning should be replaced by the `skip: True # [not py27]` from above, so I think the former was correct.